clients/src/main/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImpl.java

```diff
@@ -52,15 +52,15 @@ public class MembershipManagerImpl implements MembershipManager {
      * Member ID assigned by the server to the member, received in a heartbeat response when
      * joining the group specified in {@link #groupId}
      */
-    private String memberId;
+    private String memberId = "";
```

Comment:
Found out that the default value were incorrect.

Reply:
Did we have any test for this or do we want one?

Reply:
I found out because a few tests were broken after my change. I will take a look and to as a follow-up if it makes sense. Is it ok for you?